core/session.rb

```diff
@@ -267,11 +269,35 @@ def showBoxes
     return exit_code
   end
 
+  def showBoxField
+    box = $session.boxes.getBox($session.boxName)
+    if box == nil
+      $out.error "Box #{$session.boxName} is not found"
+      return 1
+    else
+      if $session.field != nil
+        if !box.has_key?($session.field)
+          $out.error "Box #{$session.boxName} does not have #{$session.field} key"
+          return 1
+        else
+          $out.out box[$session.field]
+          return 0
+        end
+      else
+        $out.out box
```

Comment:
Returned result is not JSON:
`{"provider"=>"virtualbox", "box"=>"bento/ubuntu-14.04", "platform"=>"ubuntu", "platform_version"=>"trusty"}`
instead of 
`{"provider":"virtualbox", "box":"bento/ubuntu-14.04", "platform":"ubuntu", "platform_version":"trusty"}`